src/common/components/Navbar/NavbarLogo.tsx

```diff
@@ -6,20 +6,20 @@ type navbarLogoProps = {
 
 const NavbarLogo: FC<navbarLogoProps> = ({ navbarLogo }) => {
   return (
-    <div className="flex mt-5 ">
-      <div className="w-[15px] h-auto bg-logoColor"></div>
-      <div className="bg-logoColor w-[180px] max-h-[100px] relative">
+    <div className="xs:w-full xs:mt-5 mt-0 flex w-2/3 sm:w-full">
+      <div className="h-auto w-[15px] bg-logoColor"></div>
+      <div className="xs:max-h-[100px] relative min-h-[80px] w-[180px] bg-logoColor">
```

Comment:
please use tailwind declared values instead of filling the code with property-[xyz]. As I said, UI doesn't have to be 1-1 with design. Please try and revise this all over the implementations.

Reply:
I modified most of them, but the one regarding the triangles made from borders are necessarily as tailwind only provides borders up to 8px